### ISSUE TITLE
Can't find Sea of Thieves

### ISSUE THREAD

**Issue by IFlickerz** [MilkyDeveloper/ReshadeForUWP#15]
Hi! I'm currently in version 2.2 and I find the game in the "Step 1.", but when it comes to "Step 2.", I just can't find the executable anywhere.

**Comment by MilkyDeveloper** [comment]
Hey! Thanks for submitting this issue. I'm a bit busy right now, so I cant fix the app itself (I probably will in the next few days :D), but on Step 2 just put click a random executable from the list. Then, just open the `.bat` file generated with Notepad and launch SoT. Open Task Manager > Details and find the SoT process name in there. Then, in the second line of the `.bat` file, you should see a command that's something like:
```
inject64.exe executable.exe | ...
```
and replace executable.exe with the process name you found in the task manager.

**Comment by IFlickerz** [comment]
I figured out using the version 1.0, but when I start my game and the Reshade loads, it just send an error that is unable to write in the Reshade.ini file.

I'll try to do what you've said and then I come here later. Many Thanks!

**Comment by IFlickerz** [comment]
Yeah, it's the same results:

![image](https://user-images.githubusercontent.com/44907434/120534615-6e7cbf00-c3b8-11eb-85d2-44286d571a2f.png)

I've already tried to edit "Reshade.ini" permissions and set them to "total control" for all users and administrators, but nothing changed. I'll try to fix it searching in reshade forums.

**Comment by IFlickerz** [comment]
I've gave permission to "ALL APPLICATIONS PACKAGE" to "ReshadeInjectUWP" and it seems to solve the "Unable to save" problem. But when I press "Home" the menu doesn't open. 😅

I forgot saying that my WindowsApps folder isn't in the same disk that Windows and ReshadeUWP is. Maybe this is the problem (I doubt).

**Comment by IFlickerz** [comment]
Yeah, I'm stuck in this problem and couldn't solve it... So my Reshade GUI doesn't open. I've tried to use another .ini that is already filled and it works a little, actually it just show the FPS on top corner, but doesn't apply any effects.
